src/pages/task/ActionButtons.tsx

```diff
@@ -245,11 +245,13 @@ export const ActionButtons = ({
           key="notifications"
           disabled={disabled}
           onClick={() => {
+            if (!isVisibleModal) {
+              taskAnalytics.sendEvent({ name: "Open Notification Modal" });
+            }
```

Comment:
What is the reasoning behind this conditional?
If the modal is open would they even be able to click on this button? Since the modal will appear over the button.

Reply:
ah good point -- I didn't consider that the 'Notify me' button would be hidden if the modal is open. I added this condition to prevent the close action from being tracked as "Open Notification Modal"

Reply:
I am still inclined to keep this condition unless we change the follow line from `setIsVisibleModal(!isVisibleModal);` to `setIsVisibleModal(true);`

Reply:
I'm for that since it stay makes it more explicit that action will open the modal

Reply:
sounds good. i made the change in d591ca7